utils/facts.js

```diff
@@ -381,5 +381,6 @@ export const facts = [
   // { fact: "", category: CATEGORIES.ANIMAL },
 ].map((f) => ({
   ...f,
+  fact: f.fact.trim(),
```

Comment:
I think it's better than clear all those extra spaces, and that won't prevent future tailing spaces entries